src/main/java/clientApplication/SimpleApplication.java

```diff
@@ -1,15 +1,20 @@
-package httpApplication;
+package clientApplication;
 
-import httpserver.HttpRequest;
-import httpserver.HttpResponse;
+import http.Application;
+import http.HttpRequest;
+import http.HttpResponse;
 
-public class SimpleApplication implements Application{
+public class SimpleApplication implements Application {
```

Comment:
Are there any tests for the `SimpleApplication`?

Reply:
Atl the tests are in spinach